packages/ui/src/app/pages/WorkingGroups/MyApplications.tsx

```diff
@@ -18,20 +19,35 @@ export const MyApplications = () => {
     ],
     []
   )
-  const { applications } = useMyApplications()
+  const { applications, isLoading } = useMyApplications()
+  const isPending = (a: WorkingGroupApplication) => a.status == 'ApplicationStatusPending'
+  const currentApplications = useMemo(() => applications?.filter(isPending), [applications])
+  const pastApplications = useMemo(() => applications?.filter((a) => !isPending(a)), [applications])
+  const pickText = () => {
+    if (isLoading) return 'Loading...'
+    return applications?.length ? 'My Applications' : 'No applications found'
+  }
 
   return (
     <AppPage crumbs={crumbs}>
       <PageHeader>
         <PageTitle>Working Groups</PageTitle>
         <WorkingGroupsTabs />
       </PageHeader>
-      <TextBig>My Applications</TextBig>
+      <TextBig>{pickText()}</TextBig>
```

Comment:
The "My applications" text was a placeholder to have _something_ on the tab displayed. You can remove it.  
  
For the loading state you can use existing component + have:

```
{loading ? (
	<Loading />
	) : (
	{currentApplications?.length ? (
          <>
            <Label>Current applications</Label>
            <ApplicationsList applications={currentApplications} />
          </>
        ) : null}
	// ... rest
) }
```